### PR TITLE
Events calendar(Issue #19)

### PR DESCRIPTION
**Pull request by steven-ferguson**
This adds a calendar for events.  https://github.com/code-scouts/green_mercury/issues/19

### CODE REVIEW

File: app/controllers/events_controller.rb [hunk 1]
```diff
@@ -24,6 +24,8 @@ def show
   end
 
   def index
+    @events_by_date = Event.all.group_by(&:date)
```
Comment (drewchandler) [inline app/controllers/events_controller.rb]: You are grabbing all the events, but only displaying a month's worth. That is quite inefficient.

File: app/controllers/events_controller.rb [hunk 2]
```diff
@@ -24,6 +24,8 @@ def show
   end
 
   def index
+    @date = params[:date] ? Date.parse(params[:date]) : Date.today
```
Comment (ErinCall) [inline app/controllers/events_controller.rb]: This needs an exception handler for invalid date params.

**Comment by steven-ferguson** [comment]
I'm reworking this to be compatible with the recent changes to Events.

**Comment by ErinCall** [comment]
This looks good to me. Have you finished your rework?

**Comment by steven-ferguson** [comment]
Yes this is good to go.

**Comment by ErinCall** [comment]
@steven-ferguson Thanks as always!

**Pull request merged**